kernel/kernel-impl/src/main/sql/mysql/sakai_content.sql

```diff
@@ -126,6 +126,7 @@ CREATE TABLE CONTENT_RESOURCE
 (
     RESOURCE_ID VARCHAR (255) NOT NULL,
     RESOURCE_UUID VARCHAR (36),
+    RESOURCE_SHA256 VARCHAR (255),
```

Comment:
please use min chars you need .... indexes get big fast

Reply:
I think 64 would always work - I just don't want to have something blow up.   When you are sure 64 will work - do you use "64" or "99"?

Reply:
char(64) or varchar(64)

Reply:
Thanks - I will use 64.